protobuf/controller/health.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package controller;
+
+option go_package = "github.com/layer5io/meshkit/protobuf/controller;controller";
+
+enum ControllerStatus {
+	RUNNING = 0;
+	IDLE = 1;
```

Comment:
unfamiliar with running vs idle - never came across idle as status in software....

Reply:
So the use case is to get the status of the pipeline defined in the controller, @mgfeller you can suggest any better language here that you think would be good to use :)